src/avtka.c

```diff
@@ -95,6 +95,25 @@ on_display(PuglView* view)
 	cairo_rectangle(cr, 0, 0, a->opts.w, a->opts.h);
 	cairo_fill(cr);
 
+	/* paint screens to the UI */
+	for (int i = 0; i < a->screen_count; i++) {
+		cairo_surface_t *s = a->screen_surfaces[i];
+		int x = a->screen_opts[i].x;
+		int y = a->screen_opts[i].y;
+#if 0
```

Comment:
?

Reply:
A method of testing that the painted surface is the shape/size as expected. This needs to be enabled as a debug option for virtual-device developers, to ensure the AVTKA screen is the size and position as expected.

Noted it should be refactored in due course, I'll probably merge with this as a TODO.